### ISSUE TITLE
Filtering of members

### ISSUE THREAD

**Issue by JRAlexander** [MicrosoftDocs/feedback#581]
Original GitHub issue [dotnet/docs#6818](https://github.com/dotnet/docs/issues/6818) from @nvmkpk:

Please let us filter listed members like hiding inherited members like MSDN website had.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb8f7ce9-5aa7-5d81-7e38-87895240399c
* Version Independent ID: 868c1749-34cc-7de8-765b-411db7f1dd19
* Content: [ToolTaskExtension Class (Microsoft.Build.Tasks)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.tasks.tooltaskextension?view=netframework-4.7.2)
* Content Source: [xml/Microsoft.Build.Tasks/ToolTaskExtension.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/Microsoft.Build.Tasks/ToolTaskExtension.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by dend** [comment]
#log-suggestion @dend

**Comment by botcrane** [comment]
🚀 **ATTENTION**: [Internal request](https://ceapex.visualstudio.com/web/wi.aspx?pcguid=7d644393-99ad-41c8-ac53-7fa79294c720&id=33900) logged.

**Comment by dotMorten** [comment]
Yes ! Soooo much this. While I appreciate documenting all members, you're quite often interested just in what a subclass adds, and there are some classes that makes it near impossible.

As one example, take a look at the following class which doesn't actually add anything over what the base class provides, but that's basically impossible to determine, because the list is so incredibly long:
https://docs.microsoft.com/en-us/dotnet/api/system.windows.documents.linebreak?view=netframework-4.7.2
